docs/usage.md

```diff
@@ -753,14 +945,72 @@ Should be a string in the format integer-unit eg. `--max_time '2.h'`
 Use to set a top-limit for the default CPU requirement for each process.
 Should be a string in the format integer-unit eg. `--max_cpus 1`
 
-### --plaintext_email
+### --single_cpu_mem
 
-Set to receive plain-text e-mails instead of HTML formatted.
+Use to set memory for a single CPU.
+Should be a string in the format integer-unit eg. `--single_cpu_mem '8.GB'`
 
-### --monochrome_logs
+## AWSBatch specific parameters
 
-Set to disable colourful command line output and live life in monochrome.
+Running the pipeline on AWSBatch requires a couple of specific parameters to be set according to your AWSBatch configuration.
+Please use [`-profile awsbatch`](https://github.com/nf-core/configs/blob/master/conf/awsbatch.config) and then specify all of the following parameters.
 
-### --multiqc_config
+### --awsqueue
 
-Specify a path to a custom MultiQC configuration file.
+The JobQueue that you intend to use on AWSBatch.
+
+### --awsregion
+
+The AWS region to run your job in.
+Default is set to `eu-west-1` but can be adjusted to your needs.
+
+### --awscli
+
+The [AWS CLI](https://www.nextflow.io/docs/latest/awscloud.html#aws-cli-installation) path in your custom AMI.
+Default: `/home/ec2-user/miniconda/bin/aws`.
+
+Please make sure to also set the `-w/--work-dir` and `--outdir` parameters to a S3 storage bucket of your choice - you'll get an error message notifying you if you didn't.
+
+## Deprecated params
+
+> :warning: These params are deprecated -- They will be removed in a future release.
```

Comment:
You've removed `:warning:` elsewhere, should do that here too - but check my commetns elsewhere.